openkfw/TruBudget
# Fix regex for finding variables in docker-compose. Minor fixes

### Checklist




### Description


### How to test

1. Set `TAG=v2.16.0` in operations/.env
2. Start `bash scripts/operation/start-trubudget.sh --full`
3. App should not fail on "STORAGE_SERVICE_PORT" not being a number
